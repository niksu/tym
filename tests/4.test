a(X, Y) :- b(X), c(Y), d().
a(X, bla) :- b(X), c(X).
b(h1).
b(h2).
c(h3).
