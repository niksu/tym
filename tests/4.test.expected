input contents |a(X, Y) :- b(X), c(Y).
a(X, bla) :- b(X), c(X).
b(h1).
b(h2).
c(h3).
|
(no query given)
stringed file contents (size=64, remaining=236)
|a(X, Y) :- b(X), c(Y)
a(X, bla) :- b(X), c(X)
b(h1)
b(h2)
c(h3).|
